backend/drm/atomic.c

```diff
@@ -44,22 +44,22 @@ static bool atomic_end(int drm_fd, struct atomic *atom) {
 }
 
 static bool atomic_commit(int drm_fd, struct atomic *atom,
-		struct wlr_drm_connector *conn, uint32_t flags, bool modeset) {
+		struct wlr_drm_backend *drm, uint32_t flags, bool modeset) {
```

Comment:
I don't think it makes sense to change this?

Reply:
Ah, you're right. I'll partly revert this, this will also keep the connector modeset information in the log.